table/block_based/block_based_table_reader.cc

```diff
@@ -542,9 +542,9 @@ Status GetGlobalSequenceNumber(const TableProperties& table_properties,
       std::array<char, 200> msg_buf;
       snprintf(
           msg_buf.data(), msg_buf.max_size(),
-          "An external sst file with version %u have global seqno property "
-          "with value %s, while largest seqno in the file is %llu",
-          version, seqno_pos->second.c_str(),
+          "An external sst file with version %u has global seqno property "
+          "with value %llu, while largest seqno in the file is %llu",
+          version, static_cast<unsigned long long>(global_seqno),
```

Comment:
This was:

Corruption: An external sst file with version 2 have global seqno property with value ³CΕ	m, while largest seqno in the file is 15351

With above change:

Corruption: An external sst file with version 2 have global seqno property with value 8117678604447195114, while largest seqno in the file is 15351709

(Underlying issue corrected in env_encrypt2.cc.)